conduit/plugins/importers/syncalgod/syncalgod_importer.go

```diff
@@ -0,0 +1,169 @@
+package syncalgod
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v3"
+
+	"github.com/algorand/indexer/conduit"
+	"github.com/algorand/indexer/conduit/plugins"
+	"github.com/algorand/indexer/conduit/plugins/importers"
+	"github.com/algorand/indexer/data"
+
+	"github.com/algorand/go-algorand-sdk/v2/client/v2/algod"
+	"github.com/algorand/go-algorand-sdk/v2/encoding/json"
+	sdk "github.com/algorand/go-algorand-sdk/v2/types"
+	"github.com/algorand/go-algorand/protocol"
+	"github.com/algorand/go-algorand/rpcs"
+)
+
+const importerName = "sync_algod"
```

Comment:
See https://github.com/algorand/indexer/pull/1431/files#diff-6efc74da31cfe4605350d8ccb909cc366261084bb90581104d5652fb494046aaR7
You need to add the import to `all.go` so it gets loaded in by the conduit command.

Reply:
We'll also want to align this name with the new name. Have you decided on Follower?

Reply:
So would we name this `algod_follower`?

Reply:
I've renamed to `algod_follower`